components/kyma-environment-broker/internal/process/deprovisioning/initialisation.go

```diff
@@ -88,7 +89,7 @@ func (s *InitialisationStep) Run(operation internal.DeprovisioningOperation, log
 func (s *InitialisationStep) run(operation internal.DeprovisioningOperation, log logrus.FieldLogger) (internal.DeprovisioningOperation, time.Duration, error) {
 	if time.Since(operation.CreatedAt) > s.operationTimeout {
 		log.Infof("operation has reached the time limit: operation was created at: %s", operation.CreatedAt)
-		return s.operationManager.OperationFailed(operation, fmt.Sprintf("operation has reached the time limit: %s", s.operationTimeout), log)
+		return s.operationManager.OperationFailed(operation, fmt.Sprintf("operation has reached the time limit: %s", s.operationTimeout), errors.New(""), log)
```

Comment:
Shouldn't there be a message/reason in the various places where the error created as `errors.New("")`?
I know it's wrapped with the original error description, but wouldn't it cause awkward displaying of the error?

Reply:
yea.. will be an extra colon in the text end... I just changed this argument to nil when there is no exact err passed in. if there is a specific err to pass in, it will be wrapped with description to keep the original err object for categorizing..